packages/wallet-management/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lifi/wallet-management",
-  "version": "2.2.0",
+  "version": "2.2.1",
```

Comment:
Why do we increment this?

Reply:
a new version would be deployed when this is merged, right?
incremented for that